bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "monocle-client-js",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
updating this version cause i already bumbed monocle-client-js previously.... do i need to do anything else?